src/StripeTests/StripeTests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
+    <Reference Include="Microsoft.CSharp" />
```

Comment:
What is this exactly?

Reply:
This is needed to use `dynamic` with .NET Framework (at least for version 4.5.2). Note that this is only added in the tests project. Users of the library who use .NET Framework and want to use `dynamic` would need to do the same thing, but the library doesn't make this a requirement.